doc/source/mergingdata.rst

```diff
@@ -72,6 +72,24 @@ In a Spatial Join, two geometry objects are merged based on their spatial relati
    cities_with_country.head()
 
 
-The ``op`` options determines the type of join operation to apply. ``op`` can be set to "intersects", "within" or "contains" (these are all equivalent when joining points to polygons, but differ when joining polygons to other polygons or lines).
+### Sjoin Arguments:
+
+``sjoin.()`` has two core arguments: ``how`` and ``op``.
+
+#### op
+The ```op`` argument specifies how ``geopandas`` decides whether or not to join the attributes of one object to another. There are three different join options as follows:
+
+* `intersects`: The attributes will be joined if the boundary and interior of the object intersect in any way with the boundary and/or interior of the other object.
+* `within`: The attributes will be join if the object’s boundary and interior intersect *only* with the interior of the other object (not its boundary or exterior).
+* `contains`: The attributes will be joined if the object’s interior contains the boundary and interior of the other object and their boundaries do not touch at all.
+
+<a href ="http://toblerity.org/shapely/manual.html#binary-predicates" target = "_blank">You can read more about each join type in the Shapely documentation.</a>
```

Comment:
The rst syntax for this is 

```
You can read more about each join type in the `Shapely documentation <http://toblerity.org/shapely/manual.html#binary-predicates>`__ 
```

(see http://www.sphinx-doc.org/en/stable/rest.html#external-links)